ProjectSettings/ProjectSettings.asset

```diff
@@ -138,11 +138,13 @@ PlayerSettings:
     16:10: 1
     16:9: 1
     Others: 1
-  bundleVersion: 2.0.0
+  bundleVersion: 1.0.0
```

Comment:
deliberate?

Reply:
build pipeline overwrites this with the data in config.cs, so I felt we should reset it?